Eyad-Mostafa.CodingTracker/README.md

```diff
@@ -0,0 +1,78 @@
+# Coding Tracker
```

Comment:
You need a section on configuring the launchSettings.json
Specifically about changing the `workingDirectory`:
```
{
  "profiles": {
    "CodingTracker": {
      "commandName": "Project",
      "workingDirectory": "C:\\Users\\Eyad Mostafa\\source\\repos\\CodingTracker\\CodingTracker"
    }
  }
}
```